test/Spec/Integration.hs

```diff
@@ -220,9 +225,30 @@ test =
           , -- Test `adjustUnbalancedTx`
             runAdjustTest
           , testBugMintAndPay
+          , runSimpleTest "MustBeSignedBy: Signed by Self and requires Others should succeed" MustBeSignedBy.testSignedBySelfAndRequiresOthers
+          , runSimpleTest "MustBeSignedBy: Signed by none and requires Others should succeed" MustBeSignedBy.testSignedByNoneAndRequireOthers
+          , runSimpleTest "VasilFeatures: Send inline datum to script" VasilFeatures.testSendInlineDatum
+          , runSimpleTest "VasilFeatures: Spend inline datum from script" VasilFeatures.testSpendInlineDatum
+          , runSimpleTest "VasilFeatures: Include reference input with inline datum in transaction with v2 script" VasilFeatures.testSpendReferenceInput
+          , runSimpleTest "VasilFeatures: Send reference script to script address" VasilFeatures.testSendReferenceScript
+          , runSimpleTest "VasilFeatures: Use reference script from script address" VasilFeatures.testSpendReferenceScript
           ]
           ++ testValueAssertionsOrderCorrectness
 
+runSimpleTest ::
+  forall (s :: Row Type) (a :: Type).
+  (Show a, Typeable a) =>
+  String ->
+  Contract.Contract Text s Text a ->
+  (TestWallets, IO (ClusterEnv, NonEmpty BpiWallet) -> TestTree)
+runSimpleTest msg contract =
+  assertExecutionWith @Text
+    [ShowTrace]
```

Comment:
Is it intentionally added to be permanent part of the test case? If I remember correctly, tracing makes test logs really verbose. Maybe this is not that crucial for plutip's own tests (when debugging is over).